.gitignore

```diff
@@ -20,4 +20,5 @@ config/database.yml
 .env
 .generators
 .byebug_history
-coverage
\ No newline at end of file
+coverage
+storage/**
```

Comment:
please check what's issue with storage files that are already included in this PR. You will need to `git delete` those.